app/web/src/newhotness/logic_composables/watched_form.ts

```diff
@@ -88,8 +84,23 @@ export const useWatchedForm = <Data>(
     const opts = formOptions({
       defaultValues: toValue(data),
     });
+
+    // keep the form up to date with values until the user
```

Comment:
I suspect we want to set dirty = false on reset too. It looks like <AttributeInput> calls `reset()` directly on the form when the input is opened back up, and we would want it to track changes until it's dirty again.